src/components/core/posts/SinglePost.tsx

```diff
@@ -82,7 +110,10 @@ export default function SinglePost({
         <div className="w-12 h-12 rounded-full border flex items-center justify-center shadow">
           <Image src="/assets/user.png" alt="user" width={40} height={40} />
         </div>
-        <span className="font-medium text-md">{singlePost && singlePost?.accountId}</span>
+        <div>
+        <h5 className="font-medium text-md">{singlePost && singlePost?.accountId}</h5>
+        {(singlePost?.$createdAt && <p className="font-thin text-xs/[10px] text-slate-950 dark:text-slate-400">{`${createdAtDateFormatter(singlePost.$createdAt)} ago`}</p>) || null}
```

Comment:
I think ternary operator will be better.

Reply:
@Sanchitbajaj02 Added ternary operator. Please check now.